pkg/codegen/python/gen.go

```diff
@@ -1933,15 +1936,35 @@ func recordProperty(prop *schema.Property, snakeCaseToCamelCase, camelCaseToSnak
 	}
 }
 
-// genInitDocstring emits the docstring for the __init__ method of the given resource type.
+// `genResourceClassDocstring` emits the class docstring for the given
+// resource type.
+//
+// Sphinx (the documentation generator that we use to generate Python
+// docs) does not draw a distinction between documentation comments on
+// the class itself and documentation comments on the `__init__`
+// method of a class. The docs repo instructs Sphinx to concatenate
+// the two together.
```

Comment:
I think any reference to how sphinx generates these things is now out of date for any of the provider SDK docs. We currently only use sphinx to generate the pulumi and pulumi-policy SDKs (which are all hand-written and unaffected by this code).

Reply:
I'll update the PR tomorrow, thank you!